www/src/components/user/EditRolesDialog.tsx

```diff
@@ -0,0 +1,48 @@
+import { DialogContent } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core';
+import { DialogContentText } from '@material-ui/core';
+import { Switch } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import React, { FC, useState } from 'react';
+
+import theme from '../../styles/theme';
+
+const EditRolesDialog: FC = () => {
+    const [isopen, setisopen] = useState(false);
+    const [isStudent, setisStudent] = useState(false);
+    const [isUser, setisUser] = useState(false);
+    const classes = useStyles();
+    // const [isopen, setisopen] = useState(false)
```

Comment:
Nit: remove comment and add a new line before the return statement

Reply:
Done